### PR TITLE
Fix/Enhance Note and Note Row Editor

### PR DESCRIPTION
**Pull request by seangoodvibes**
Fixes / enhancements to note and note row editors

- Disable affect entire when entering the kit menu's
- Enable toggling of affect entire when in the kit note row menu
- Enable toggling of playback when in the menu's
- Enable toggling of fill when in the menu's
- Enable toggling of stutter
- Enable toggling of shift
- Enable note editing with shift
- Enable editing note probability, iteration, fill with shift
- Enable note nudging in the row editor
- Enable note repeat editing in the row editor
- Enable note velocity editing in the row editor
- Enable horizontal scrolling in the row editor
- Enable zooming by holding audition pad + pressing and turning horizontal encoder
- Enable editing note row length by holding audition pad + turning horizontal encoder
- Enable rotating note row when shift is enabled by holding audition pad + pressing and turning horizontal encoder
- Fix big "T" font that was rendering an extra pixel

To do in follow-up PR:
- [ ] Hold Affect Entire to Change All Notes in a Step
- [ ] Hold Affect Entire + Clip to Change All Notes in a Clip
- [ ] Row Velocity Editing

### CODE REVIEW

File: src/deluge/gui/fonts/fonts.c [hunk 1]
```diff
@@ -2681,7 +2681,7 @@ const uint8_t font_metric_bold_20px[] = {
         0b00001111, 0b00000000, 0b00000000,
         0b00001111, 0b00000000, 0b00000000,
         0b00001111, 0b00000000, 0b00000000,
-        0b00011111, 0b00000000, 0b00000000,
```
Comment (m-m-adams) [inline src/deluge/gui/fonts/fonts.c:2684]: what's this?

File: src/deluge/gui/l10n/english.json [hunk 2]
```diff
@@ -875,15 +875,18 @@
         "STRING_FOR_CLIP_NAME": "Clip Name",
         
         "STRING_FOR_NOTE_EDITOR": "Note Editor",
-        "STRING_FOR_NOTE_EDITOR_VELOCITY": "Velocity",
-        "STRING_FOR_NOTE_EDITOR_PROBABILITY": "Probability",
-        "STRING_FOR_NOTE_EDITOR_ITERANCE": "Iterance",
-        "STRING_FOR_NOTE_EDITOR_FILL": "Fill",
+        "STRING_FOR_NOTE_EDITOR_VELOCITY": "Note Velocity",
```
Comment (m-m-adams) [inline src/deluge/gui/l10n/english.json:878]: Do we need these? I think it's fairly clear which velocity/probability you're editing from context, and strings eat into our ram (until someone figures out how to move them to external ram)

File: src/deluge/gui/fonts/fonts.c (hunk 1 above)
Comment (seangoodvibes) [reply src/deluge/gui/fonts/fonts.c:2684]: Fixed an erroneous pixel on the T font

It looked like this

.......
 ....
  ...
  ...

File: src/deluge/gui/l10n/english.json (hunk 2 above)
Comment (seangoodvibes) [reply src/deluge/gui/l10n/english.json:878]: I did this for when you're holding a note in the row editor and the display refreshes to show the value of that note (so it switches  from saying row probability to note probability).

If you don't think it's worth the extra strings I can remove them

**Pull request merged**